tests/foundry/common/documents/actor.test-d.ts

```diff
@@ -68,7 +70,7 @@ class MyCharacter extends foundry.abstract.TypeDataModel<MyCharacterSchema, Acto
     };
   }
 
-  prepareDerivedData(): void {
+  prepareDerivedData(this: Merge<DataModel<MyCharacterSchema, Actor.ConfiguredInstance>, {}>): void {
```

Comment:
This change I made and forgot to revert is wrong, even if it's just some tests, leaving it owuld probably cause confusions in the future.